lib/auth_web/controllers/auth_controller.ex

```diff
@@ -105,10 +140,27 @@ defmodule AuthWeb.AuthController do
   def google_handler(conn, %{"code" => code, "state" => state}) do
     {:ok, token} = ElixirAuthGoogle.get_token(code, conn)
     {:ok, profile} = ElixirAuthGoogle.get_user_profile(token.access_token)
-
     # save profile to people:
     person = Person.create_google_person(profile)
 
+    # login log
+    user_agent =
+      conn
+      |> get_user_agent_string()
+      |> Auth.UserAgent.get_or_insert_user_agent()
+
+    ip_address = get_ip_address(conn)
+
+    Auth.LoginLog.create_login_log(
+      %{
+        email: person.email,
+        ip_address: ip_address,
+        auth_provider: "google"
```

Comment:
Again, let Google handle the failed login attempts for their Auth system. 
we only care about `email+password` for our logs.
We will have `sessions` for tracking who signed in with which Auth provider. #30